### PR TITLE
[CodeCamp2023-313]ExLPose data set support

### PR DESCRIPTION
**Pull request by Yang-Changhui**
## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Comment by Ben-Louis** [comment]
Hi @Yang-Changhui, thank you for your contribution. Could you please add description of ExLPose in `docs/en/dataset_zoo/2d_body_keypoint.md`?

**Comment by Yang-Changhui** [comment]
@Ben-Louis 
image
Can you tell me how to generate a picture like this?

**Comment by Tau-J** [comment]
> @Ben-Louis image Can you tell me how to generate a picture like this?

This skeleton picture is drawn via [draw.io](draw.io)

File: docs/en/dataset_zoo/2d_body_keypoint.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ MMPose supported datasets:
   - [OCHuman](#ochuman) \[ [Homepage](https://github.com/liruilong940607/OCHumanApi) \]
   - [MHP](#mhp) \[ [Homepage](https://lv-mhp.github.io/dataset) \]
   - [Human-Art](#humanart) \[ [Homepage](https://idea-research.github.io/HumanArt/) \]
+  - [ExLPose](#exlpose-dataset)\[ [Homepage](http://cg.postech.ac.kr/research/ExLPose/) \]
```
Comment (Ben-Louis) [inline docs/en/dataset_zoo/2d_body_keypoint.md:17]:
```suggestion
  - [ExLPose](#exlpose-dataset) \[ [Homepage](http://cg.postech.ac.kr/research/ExLPose/) \]
```

File: docs/en/dataset_zoo/2d_body_keypoint.md [hunk 2]
```diff
@@ -586,3 +585,51 @@ mmpose
...
+        	|	|-- ExLPose
+                │   │	|-- ExLPose_train_LL.json
+                │   |	|-- ExLPose_test_LL-A.json
+                │   │	|-- ExLPose_test_LL-E.json
+                │   |	|-- ExLPose_test_LL-H.json
+                │   │	|-- ExLPose_test_LL-N.json
+        |-- dark
+            |	|--00001.png
+            |	|--00002.png
+            |	|--...
```
Comment (Ben-Louis) [inline docs/en/dataset_zoo/2d_body_keypoint.md:633]:
```suggestion
        │-- annotations
        |   │-- ExLPose
        |       │-- ExLPose_train_LL.json
        |       │-- ExLPose_test_LL-A.json
        |       │-- ExLPose_test_LL-E.json
        |       │-- ExLPose_test_LL-H.json
        |       │-- ExLPose_test_LL-N.json
        |-- dark
            │-- 00001.png
            │-- 00002.png
            │-- ...
```

File: mmpose/datasets/datasets/body/exlpose_dataset.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+@DATASETS.register_module()
+class ExlposeDataset(BaseCocoStyleDataset):
+    """Exlpose dataset for pose estimation.
+
+    "Human Pose Estimation in Extremely Low-Light Conditions",
+    arXiv'2023.
+    More details can be found in the `paper
+    <https://arxiv.org/abs/2303.15410>`__
+
+    AIC keypoints:
```
Comment (Ben-Louis) [inline mmpose/datasets/datasets/body/exlpose_dataset.py:15]: Please verify if the description provided in the docstring aligns with the dataset

File: mmpose/datasets/datasets/body/exlpose_dataset.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+@DATASETS.register_module()
+class ExlposeDataset(BaseCocoStyleDataset):
+    """Exlpose dataset for pose estimation.
+
+    "Human Pose Estimation in Extremely Low-Light Conditions",
+    CVPR'2023.
+    More details can be found in the `paper
+    <http://cg.postech.ac.kr/research/ExLPose/>`__.
+
+    AIC keypoints:
```
Comment (Ben-Louis) [inline mmpose/datasets/datasets/body/exlpose_dataset.py:15]:
```suggestion
    ExLPose keypoints:
```

**Review by Ben-Louis — APPROVED** [review]

**Pull request merged**